browser/components/MarkdownEditor.js

```diff
@@ -8,6 +8,10 @@ class MarkdownEditor extends React.Component {
   constructor (props) {
     super(props)
 
+    this.hotkey = props.config.hotkey
+
+    this.keyPressed = []
```

Comment:
I feel `keyPressed` should be at `state` however I could not write smart code by using state. I want an advice about it.